packages/datadog-plugin-next/test/index.spec.js

```diff
@@ -89,11 +96,64 @@ describe('Plugin', function () {
         const files = [
           'package.json',
           'node_modules',
-          '.next'
+          '.next',
+          'yarn.lock'
         ]
         const paths = files.map(file => `${__dirname}/${file}`)
         execSync(`rm -rf ${paths.join(' ')}`)
       })
+    }
+
+    const initStandaloneFiles = () => {
```

Comment:
There is a lot happening in this function. Is it all really needed?

Reply:
> I can fix it there if that's needed

I would say yes let's do it, because otherwise it would mean our support is actually currently incomplete if I understand this correclty.